### PR TITLE
PR: PixelData test failing due to temporary file move

### PR DESCRIPTION
**Pull request by hsautessella**
A couple of `PixelData` tests have been updated to look for pixel temporary files in the correct place.

Addresses #538

### CODE REVIEW

**Review by oerc0122 — APPROVED** [review]

File: _test/test_sqw/test_PixelData.m [hunk 1]
```diff
@@ -597,7 +597,10 @@ function test_dirty_pix_tmp_files_are_deleted_when_pix_out_of_scope(obj)
             old_rng_state = rng();
             fixed_seed = 774015;
             rng(fixed_seed, 'twister');  % this seed gives an expected object_id_ = 54452
-            expected_tmp_dir = fullfile(tempdir(), 'sqw_pix54452');
+            expected_tmp_dir = fullfile( ...
+                get(parallel_config, 'working_directory'), ...
+                'sqw_pix54452' ...
+            );
             clean_up = onCleanup(@() rng(old_rng_state));
```
Comment (cmarooney-stfc) [inline _test/test_sqw/test_PixelData.m:604]: Shouldn't the onCleanup call precede the rng call on L.599? Ditto at L.899

Comment (nickbattam-tessella) [reply _test/test_sqw/test_PixelData.m:604]: They will work fine where they are: the `old_rng_state` variable is set as the rng is reseeded. This call creates function pointer that is executed at the end of the method to restore that.

**Review by nickbattam-tessella — APPROVED** [review]

File: _test/test_sqw/test_PixelData.m (hunk 1 above)
Comment (cmarooney-stfc) [reply _test/test_sqw/test_PixelData.m:604]: @nickbattam-tessella What if rng or fullfile errors?

Comment (hsautessella) [reply _test/test_sqw/test_PixelData.m]: I've moved the call up 👍

**Pull request merged**